search-app/test/cmr/search/test/data/metadata_retrieval/metadata_transformer.clj

```diff
@@ -1,15 +1,11 @@
 (ns cmr.search.test.data.metadata-retrieval.metadata-transformer
   (require
    [clojure.test :refer :all]
-   [cmr.common.time-keeper :as tk]
    [cmr.common.util :as util]
-   [cmr.search.data.metadata-retrieval.metadata-transformer :as mt]
-   [cmr.search.test.data.metadata-retrieval.test-metadata :as tm]
-   [cmr.umm-spec.test.expected-conversion :as expected-conversion]))
+   [cmr.search.data.metadata-retrieval.metadata-transformer :as metadata-transformer]
+   [cmr.search.test.data.metadata-retrieval.test-metadata :as tm]))
 
-(use-fixtures :each tk/freeze-resume-time-fixture)
-
-(def original-transform-strategy mt/transform-strategy)
+(def original-transform-strategy metadata-transformer/transform-strategy)
 
 (deftest transform-to-multiple-formats-test
```

Comment:
The original use of fixture to freeze time in time-keeper in this test causes timekeeper in revision_format_map test to reset incorrectly due to the exception handling. I chose to work around this problem by removing the timekeeper fixture as the metadata checking in the original test is kind of an overkill.